Source/DafnyRuntime/DafnyRuntime.h

```diff
@@ -80,6 +80,16 @@ void dafny_print(T x) {
   std::cout << x;
 }
 
+// Special-case bool so that the C++ output matches that of other backends
+template<>
+void dafny_print<bool>(bool x) {
+  if (x) {
+    std::cout << "true";
+  } else {
+    std::cout << "false";
+  }
```

Comment:
```suggestion
  std::cout << (x ? "true" : "false");
```
?

Reply:
Or the "std' way: std::cout << std::boolalpha << x

Reply:
Good point -- I'd forgotten about that option.  That one probably requires a comment for the uninitiated.  I don't feel strongly between the two versions.